qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -249,7 +266,7 @@ def run(
 
         # Configure run options for simulation
         if options:
-            backend = copy.copy(self)
+            backend = copy.deepcopy(self)
```

Comment:
What's the reason for this change? Is it because the newly added fields contain nested mutable objects?

Reply:
So, when I do, `job1 = backend.run(circuits, meas_level=1)` followed by `job2 = backend.run(circuits)` it continues to give `meas_level=1` data. A real backend does not have this behaviour. Options given to `run` only affect that particular job.